teos/extended_appointment.py

```diff
@@ -2,13 +2,33 @@
 
 
 class ExtendedAppointment(Appointment):
-    def __init__(self, locator, to_self_delay, encrypted_blob, user_id):
-        super().__init__(locator, to_self_delay, encrypted_blob)
+    """
+    The :class:`ExtendedAppointment` contains extended information about an appointment between a user and the tower.
+
+    While it shares some information with an :class:`Appointment <common.appointment.Appointment>`, it also
```

Comment:
"it shares some information" seems to suggests that neither class is subclass of the other. I'd say instead "It extends the Appointment with information relevant to the tower, such as..."